### DISCUSSION TITLE
Add support for changing contact list width

### DISCUSSION THREAD

**Discussion by luntik2012** [d99kris/nchat#80] [Ideas]
Contact names do not fit into the contact list which has hardcoded width (14). Suggestion is to add the ability to change the width of the list.

I didn't notice CONTRIBUTING.md at first and already [implemented](https://github.com/d99kris/nchat/compare/master...luntik2012:nchat:master) this in a fork and can create PR if everything is ok.

**Comment by d99kris** [comment]
Hi @luntik2012 - thanks for suggesting the feature and preparing an implementation. I think it's a reasonable feature given other UI elements can be disabled/enabled at run-time. The implementation looks good at a first glance, but I'll need to review it more in detail before merging. 

I will proceed to move this `discussion` to an `issue` (= an accepted feature request). 

Yes, please create a PR and I hope I'll get to review / test it out it within the next week or so.

**Comment by d99kris** [comment]
Please reference issue #81 in the PR title, like "fixes #81 - add support for changing contact list width".
